### PR TITLE
Add: japanese translate

### PR DESCRIPTION
**Pull request by tuna2134**

### CODE REVIEW

**Comment by tuna2134** [comment]
@ahopkins I created new.

**Comment by miss85246** [comment]
Just now, I have a brief look at your translated documents. I saw that you still have a lot of documents that haven't been translated. I don't think the current translation completion degree is suitable for merge. Can you try to translate more documents? :handshake:

**Comment by tuna2134** [comment]
Yeah, and I request to help a translate.

**Comment by tuna2134** [comment]
Ok
I translated finish deployment.

**Comment by ahopkins** [comment]
How's this going? Do you need any help? Any questions?

**Comment by tuna2134** [comment]
Oh…
I'm alright。

**Comment by tuna2134** [comment]
@miss85246 Now finish

**Comment by ahopkins** [comment]
@tuna2134 :clap: 

Sounds great! Make sure to pull the latest changes from `main` into the PR and resolve any conflicts. Then we can get this merged and built.

**Comment by tuna2134** [comment]
Oh..

**Comment by tuna2134** [comment]
> 

For example please

**Comment by ahopkins** [comment]
image

**Comment by tuna2134** [comment]
Oh…

**Comment by ahopkins** [comment]
It looks like you made some changes to the config, which is now broken out into multiple files. So, you just need to add your changes to the `locales` files instead.

**Comment by tuna2134** [comment]
How to fix?

**Comment by tuna2134** [comment]
@ahopkins finish

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 name: Build and Deploy
 on:
-  push:
-    branches:
-      - main
+  release:
+    types: [created]
```
Comment (ahopkins) [inline .github/workflows/main.yml:4]: Why this change? Then we need to create a release. Merging to `main` seems simple enough for our current use case. Am I missing somehting?

File: src/.vuepress/config.js [hunk 2]
```diff
@@ -40,6 +341,7 @@ let site_config = {
   locales: {
     "/en/": locales.locale_abstract_en,
     "/zh/": locales.locale_abstract_zh,
+    "/ja/": ja,
     "/ko/": locales.locale_abstract_ko,
   },
```
Comment (ahopkins) [inline src/.vuepress/config.js:346]: Please follow the existing pattern and do not add more than is necessary to this file. These should be loaded by reference, not to local values.

File: src/ja/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+home: true
+heroImage: https://raw.githubusercontent.com/huge-success/sanic-assets/master/png/sanic-framework-logo-400x97.png
```
Comment (ahopkins) [inline src/ja/README.md:3]: Please update `huge-success` to `sanic-org`

**Review by ahopkins — CHANGES_REQUESTED** [review]
A couple of housekeeping items. Thanks for the effort on this.

File: .github/workflows/main.yml (hunk 1 above)
Comment (tuna2134) [reply .github/workflows/main.yml:4]: I used it to move it temporarily.

**Comment by tuna2134** [comment]
@ahopkins finished

File: src/.vuepress/locales/details/locale_detail_ja.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+const nav_ja = require("./nav/nav_ja");
+
+let ja_locales = {
+   selectText: "Languages",
+   label: "Japanese",
+   editLinkText: "Edit this page on GitHub",
+   algolia: {},
+   nav: nav_ja,
+   sidebar: sidebar_ja,
+};
```
Comment (ahopkins) [inline src/.vuepress/locales/details/locale_detail_ja.js:11]: Can you make sure all of these fields are covered:

```js
module.exports = {
  selectText: "Languages",
  label: "English",
  editLinkText: "Edit this page on GitHub",
  helpText: "Help",
  algolia: {},
  current: "Current with version 21.12",
  nav: nav_en,
  sidebar: sidebar_en,
};
```

Comment (ahopkins) [reply src/.vuepress/locales/details/locale_detail_ja.js:11]: Looks like `helpText` and `current` are missing.

**Comment by tuna2134** [comment]
Fix

**Review by ahopkins — APPROVED** [review]

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -1,8 +1,8 @@
 name: Build and Deploy
 on:
   push:
-    branches:
-      - main
+   branches:
+     - main
```
Comment (ahopkins) [inline .github/workflows/main.yml:5]: One more item. Can you back out this change?

**Review by ahopkins — CHANGES_REQUESTED** [review]

**Comment by tuna2134** [comment]
Ok

**Comment by tuna2134** [comment]
Fixed

**Comment by tuna2134** [comment]
@ahopkins Is this ok?

**Review by ahopkins — APPROVED** [review]

**Comment by tuna2134** [comment]
Oh
finished!
